### PR TITLE
Updates Fritz2 community blurb.

### LINKED ISSUE #3149: Bug - Community page typo & broken link (Fritz2/Kotlin)
**Describe the problem**
On the [PatternFly Community page](https://www.patternfly.org/v4/community), several updates are needed to the Fritz2 community blurb: 
- Update the community project name from "PatternFly Fritz2" to "PatternFly Kotlin." 
- Update the title and showcase links. 
- Revise the copy.

**Expected behavior**
- **New name**: PatternFly Kotlin
- **New link for title**: https://github.com/patternfly-kotlin/patternfly-kotlin
- **New link for showcase**: https://patternfly-kotlin.github.io/patternfly-kotlin-showcase/#home 
- **Revised copy**:  
PatternFly Kotlin is a Kotlin implementation of PatternFly based on fritz2, targeting Kotlin/JS. The goal of this project is to provide all PatternFly components in Kotlin, matching the reactive nature of fritz2. The components use stores, handlers, and other elements from the fritz2 API. For a quick overview, check out the [PatternFly Kotlin showcase.](https://patternfly-kotlin.github.io/patternfly-kotlin-showcase/#home)

**Any other information?** 
The contributor of Kotlin is Harald Pehl (Red Hatter).

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #3149

### CODE REVIEW

**Review by abigaeljamie — APPROVED** [review]

File: packages/v4/patternfly-docs/pages/community.js [hunk 1]
```diff
@@ -85,8 +85,8 @@ const CommunityPage = () => {
...
-      <p>PatternFly Fritz2 is a Kotlin implementation of PatternFly based on fritz2 targeting Kotlin/JS. The goal of this project is to provide all PatternFly components in Kotlin. This is done in a way that matches the reactive nature of fritz2. In particular, the components use stores, handlers, and other elements from the fritz2 API. To get a quick overview what this is all about head over to the <a href="https://patternfly-kotlin.github.io/patternfly-fritz2-showcase/#home">PatternFly Fritz2 showcase</a>.</p>
+      <Title size="xl" className="pf-u-mb-sm ws-page-title pf-u-mt-xl" headingLevel="h3"><a href="https://github.com/patternfly-kotlin/patternfly-kotlin">PatternFly Kotlin</a></Title>
+      <p>PatternFly Kotlin is a Kotlin implementation of PatternFly based on fritz2, targeting Kotlin/JS. The goal of this project is to provide all PatternFly components in Kotlin, matching the reactive nature of fritz2. The components use stores, handlers, and other elements from the fritz2 API. For a quick overview, check out the <a href="https://patternfly-kotlin.github.io/patternfly-kotlin-showcase/#home">PatternFly Kotlin showcase</a>.</p>
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/pages/community.js:89]: last link is still broken!

**Comment by mmenestr** [comment]
Should the mentions of "Frizt2" be title cased instead of sentence case? Since it's title case in the header? Just double checking

File: packages/v4/patternfly-docs/pages/community.js (hunk 1 above)
Comment (abigaeljamie) [reply packages/v4/patternfly-docs/pages/community.js:89]: Weird, it works for me! How are you getting to the broken link? I'm putting the URL in a new window, and it brings me to the Kotlin/Fritz2 landing page.

Comment (abigaeljamie) [reply packages/v4/patternfly-docs/pages/community.js:89]: It looks like the preview never updated....

**Comment by abigaeljamie** [comment]
> 
Good question! I looked it up, and I see it written in lowercase across Google sources, so I left it as-is. I think the only reason it was capitalized before is because it was the name of the community.

**Pull request merged**